semgrep-core/src/matching/Generic_vs_generic.ml

```diff
@@ -126,15 +126,21 @@ let rec obj_and_method_calls_of_expr e =
   match e.G.e with
   | B.Call ({ e = B.DotAccess (e, tok, fld); _ }, args) ->
       let o, xs = obj_and_method_calls_of_expr e in
-      (o, (fld, tok, args) :: xs)
+      (o, (fld, tok, Some args) :: xs)
+  | B.DotAccess (e, tok, fld) ->
```

Comment:
nice, I wrote the original code, and I'm surprised the fix was so simple :) I imagined worse.